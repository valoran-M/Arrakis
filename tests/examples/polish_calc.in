dune exec arrakis -- ./examples/polish_calc.s --no-color --run

1
2 + p
2 * p
1 - p
q

