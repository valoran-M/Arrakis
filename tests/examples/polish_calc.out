3
6
5
Info: Exiting with code '0'
